about_regular_expressions.exs

```diff
@@ -0,0 +1,36 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = Regex.compile!("This [W|w]orks!")
```

Comment:
That sounds like a good idea. I am currently trending towards have an "about sigils" lesson and then just using the sigils from the get go in the regular expressions lessons without making a big fuss about them.